packages/eslint-plugin-material-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-material-ui",
-  "version": "5.0.0-alpha.33",
+  "version": "5.0.0-alpha.35",
```

Comment:
should it be increment by 1 => `5.0.0-alpha.34`?

I have no strong opinion here since it is still in alpha, but I lean toward whatever `lerna version` gave us. Last time that I did the release, I feel like there will be some error prone if we make the change manually.

If we want the version to be the same, then we should use `independent: false`

Reply:
The above version is expected.

We have configured `lerna version` for the version to [be independent](https://github.com/mui-org/material-ui/blob/3e25a2de5d8156827bf6b82107684d6e3ac591f6/lerna.json#L4) because what "framer" needed was unclear. But the target is

```diff
diff --git a/lerna.json b/lerna.json
index 06cff9cd69..8a385eae09 100644
--- a/lerna.json
+++ b/lerna.json
@@ -1,5 +1,5 @@
 {
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "independent"
+  "version": "5.0.0-alpha.35"
 }
```

I think that we should aim for the version to be as close as possible when in the same ["realm"](https://github.com/mui-org/material-ui/pull/26477#issuecomment-851059745). This makes it easier for the developers to know the manage versions. 1. At a glance at the package.json/yarn.lock you know if you the versions are compatible. 2. Our approach to version the documentation heavily depends on having a single version.